src/EnergyPlus/WindowManagerExteriorOptical.cc

```diff
@@ -249,13 +249,16 @@ namespace Window {
                     constrSh.effShadeBlindEmi[iSlatAng] = EpsShIR * (1.0 + RhoGlIR * TauShIR / (1.0 - RhoGlIR * RhoShIR));
                     constrSh.effGlassEmi[iSlatAng] = EpsGlIR * TauShIR / (1.0 - RhoGlIR * RhoShIR);
                 }
-
-                surfShade.effShadeEmi = Interp(constrSh.effShadeBlindEmi[surfShade.blind.slatAngIdxLo],
-                                               constrSh.effShadeBlindEmi[surfShade.blind.slatAngIdxHi],
-                                               surfShade.blind.slatAngInterpFac);
-                surfShade.effGlassEmi = Interp(constrSh.effGlassEmi[surfShade.blind.slatAngIdxLo],
-                                               constrSh.effGlassEmi[surfShade.blind.slatAngIdxHi],
-                                               surfShade.blind.slatAngInterpFac);
+                // The following surface assignments previously lived in the WindowManagerExteriorThermal file.
+                // They were moved here during the Material refactor, however I found that the array index values Lo/Hi
+                // were making it here still as -1, and this was causing a difficult to diagnose segfault.
+                // I'm moving these back over there, but open to suggestion if they need to be here.
+                // surfShade.effShadeEmi = Interp(constrSh.effShadeBlindEmi[surfShade.blind.slatAngIdxLo],
+                //                                constrSh.effShadeBlindEmi[surfShade.blind.slatAngIdxHi],
+                //                                surfShade.blind.slatAngInterpFac);
+                // surfShade.effGlassEmi = Interp(constrSh.effGlassEmi[surfShade.blind.slatAngIdxLo],
+                //                                constrSh.effGlassEmi[surfShade.blind.slatAngIdxHi],
+                //                                surfShade.blind.slatAngInterpFac);
```

Comment:
The comment explains it, but basically, on my Apple Silicon, in a release build, this was throwing a segfault because the array indices were still -1 here.  I just moved them to where they were assigned before the Material refactor, and it's all happy locally.  We'll see how CI likes it.